tekton/utils/pipeline_run_builder.go

```diff
@@ -111,6 +112,22 @@ func (b *PipelineRunBuilder) WithObjectReferences(objects ...client.Object) *Pip
 	return b
 }
 
+// WithTenantArrayPrams constructs tektonv1.Param entries for each of the provided client.Objects.
+// Each param name is modified to have the prefix "tenant".
```

Comment:
Why? These are for tenant defined pipelines. Why would we modify the parameters they pass to their own pipelines, so they have to modify their pipeline definitions to fit?

Reply:
From my understanding, the user could somehow also add some extra parameters in the releasePlan itself. I also added the `tenant` prefix as the parameter name might collide with some previous existent one, *release* for example. This is up for discussion - maybe this is not required at the moment, or some misunderstanding from my side. Can you comment here @davidmogar ?

Reply:
I am for now removing the _tenant_ prefix as what you said makes sense - although we might need to have some check to display a clear error message in case the user adds a parameters that collides with a existent one. Anyhow, lets hear from David tomorrow.

Reply:
Said check can always come in a future PR once a user hits it and raises it